apps/prairielearn/src/lib/sentry.ts

```diff
@@ -5,7 +5,7 @@ import * as Sentry from '@prairielearn/sentry';
 export function enrichSentryEventMiddleware(req: Request, res: Response, next: NextFunction) {
   // This will ensure that this middleware is always run in an isolated
   // context so that we don't accidentally leak data between requests.
-  Sentry.runWithAsyncContext(() => {
+  Sentry.withScope(() => {
     const scope = Sentry.getCurrentScope();
```

Comment:
I believe, from the docs, that you can do this:
```suggestion
  Sentry.withScope((scope) => {
```

Reply:
Done in 2d746a2.